### PR TITLE
NAVQP-160: add U-Boot env tools to the desktop image

### PR DESCRIPTION
**Pull request by avyurtsev**

### CODE REVIEW

File: recipes-navq/u-boot-env-setup/u-boot-env-setup.bb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	echo "/dev/mmcblk2	0x400000	0x4000" > ${D}${sysconfdir}/fw_env.config_emmc
+
+	install -m 0755 ${WORKDIR}/u-boot-env-setup.sh ${D}${sbindir}
+	install -m 0644 ${WORKDIR}/u-boot-env-setup.service ${D}${systemd_system_unitdir}
+}
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYSTEMD_SERVICE:${PN} = "u-boot-env-setup.service"
+FILES:${PN} += "${systemd_system_unitdir}"
+INSANE_SKIP:${PN} += "file-rdeps"
```
Comment (db-voxel) [inline recipes-navq/u-boot-env-setup/u-boot-env-setup.bb]: Is this really needed?

**Review by sposelenov — APPROVED** [review]
@avyurtsev Looks good. Did you test it?

File: recipes-navq/u-boot-env-setup/u-boot-env-setup.bb (hunk 1 above)
Comment (avyurtsev) [reply recipes-navq/u-boot-env-setup/u-boot-env-setup.bb]: You mean insane skip? bitbake detects that the shell script is using bash, and wants that dependency to be set in the bb script. But the bash package comes from the Ubuntu layer, and cannot be listed as RDEPENDS. So yes, we need insane_skip to get rid of the error with bash.

**Comment by avyurtsev** [comment]
> @avyurtsev Looks good. Did you test it?

Sure, booted from EMMC and SD, the link to fw_env is correctly updated in both cases.

**Review by db-voxel — APPROVED** [review]

File: recipes-navq/u-boot-env-setup/u-boot-env-setup.bb (hunk 1 above)
Comment (avyurtsev) [reply recipes-navq/u-boot-env-setup/u-boot-env-setup.bb]: Ok, after last conversation I've removed dependency on bash. So no need for insane_skip anymore. Please take a loook.

**Pull request merged**